_po/ja/how-to/laravel.po

```diff
@@ -304,18 +441,33 @@ msgid ""
 "};\n"
 "```"
 msgstr ""
+"    /**\n"
+"     * Reverse the migrations.\n"
+"     */\n"
+"    public function down(): void\n"
+"    {\n"
+"        DB::statement('DROP INDEX pgrn_posts_id_index');\n"
+"        DB::statement('DROP INDEX pgrn_posts_title_index');\n"
+"        DB::statement('DROP INDEX pgrn_posts_body_index');\n"
+"        DB::statement('DROP EXTENSION IF EXISTS pgroonga');\n"
+"        Schema::dropIfExists('posts');\n"
+"    }\n"
+"};\n"
+"```"
 
 msgid ""
 "#### Controller\n"
 "Let’s add our Post Controller to handle user requests `app/Http/Controller/Pos"
 "tController.php`"
 msgstr ""
+"#### コントローラ\n"
+"リクエストを捌くPostコントローラ`app/Http/Controller/PostController.php`を作ります。"
 
 msgid "namespace App\\Http\\Controllers;"
-msgstr ""
+msgstr "namespace App\\Http\\Controllers;"
 
 msgid "use Illuminate\\Http\\Request;"
-msgstr ""
+msgstr "use Illuminate\\Http\\Request;"
```

Comment:
Probably, this is needless.
Because this test is same as before translation text.